project/src/main/webapp/index.html

```diff
@@ -3,15 +3,16 @@
   <head>
     <meta charset="utf-8" />
     <title>Collaborative Text Editor</title>
-    <link rel="stylesheet" href="style.css" />
-    <script src="script.js"></script>
     <script src="https://www.gstatic.com/firebasejs/5.5.4/firebase.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/javascript/javascript.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/python/python.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.css" />
     <link rel="stylesheet" href="https://firepad.io/releases/v1.5.9/firepad.css" />
+    <link rel="stylesheet" href="https://codemirror.net/theme/ayu-dark.css" />
     <script src="https://firepad.io/releases/v1.5.9/firepad.min.js"></script>
-
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js"></script>
     <style>
```

Comment:
Is there a particular reason that you have all this styling in the HTML file and not in a separate stylesheet?